python/cc/BUILD.bazel

```diff
@@ -0,0 +1,44 @@
+# Package for C/C++ specific functionality of the Python rules.
+
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("//python/private:current_py_cc_headers.bzl", "current_py_cc_headers")
```

Comment:
We need to file and issue to use `//` instead of `@` when using rules Python internally.

Reply:
Filed https://github.com/bazelbuild/rules_python/issues/1296